### ISSUE TITLE
convert_glatos_to_att GDAL warning/expected output

### ISSUE THREAD

**Issue by mhpob** [ocean-tracking-network/glatos#198]
Running the test in test-covert_glatos_to_att:

```
  wd_file <- system.file("extdata",
                         "walleye_detections.csv",
                         package = "glatos"
  )

  wald <- read_glatos_detections(wd_file)

  # get path to example receiver file
  rec_file <- system.file("extdata", "sample_receivers.csv",
                          package = "glatos"
  )
  recd <- read_glatos_receivers(rec_file) # load receiver data

  watt <- convert_glatos_to_att(wald, recd)
```

The marinespecies.org link prints to console, but is not captured:
`[1] "https://www.marinespecies.org/rest/AphiaRecordsByVernacular/walleye"`
[Seems it's from a call to `convert_otn_to_att`](https://github.com/ocean-tracking-network/glatos/blob/c6ca9c1f3e540b2ddfbe6e99899b7a09f211405d/R/util-convert_otn_to_att.r#L223). Is this expected behavior for convert_glatos_to_att?

Also GDAL throws a warning [due to the default PROJ string](https://github.com/ocean-tracking-network/glatos/blob/dev/R/util-convert_glatos_to_att.r#L44).

`Warning message:
In CPL_crs_from_input(x) :
  GDAL Message 1: +init=epsg:XXXX syntax is deprecated. It might return a CRS with a non-EPSG compliant axis order.`

Could possibly fix by supplying the whole proj4 string: `'+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs'

**Comment by chrisholbrook** [comment]
GDAL warning fixed in commit deeadaf

**Comment by chrisholbrook** [comment]
> The marinespecies.org link prints to console, but is not captured:
> `[1] "https://www.marinespecies.org/rest/AphiaRecordsByVernacular/walleye"`
> [Seems it's from a call to `convert_otn_to_att`](https://github.com/ocean-tracking-network/glatos/blob/c6ca9c1f3e540b2ddfbe6e99899b7a09f211405d/R/util-convert_otn_to_att.r#L223). Is this expected behavior for convert_glatos_to_att?

fixed in commit 619795e.
